rom/dev/src/flow/cold_reset/fmc_alias.rs

```diff
@@ -313,6 +313,27 @@ impl FmcAliasLayer {
             manifest.fmc.load_addr,
             manifest.fmc.size
         );
+        cprintln!(
+            "[afmc] Loading Runtime at address 0x{:08x} len {}",
+            manifest.runtime.load_addr,
+            manifest.runtime.size
+        );
+
+        // Detect if FMC and RT address ranges overlap
+        // Load address of FMC <= End address of RT
+        // and End address of FMC >= Load address of RT
+        if manifest.fmc.load_addr <= manifest.runtime.load_addr + manifest.runtime.size
```

Comment:
is this first check really required and if so, shall we also check the dual condition fmc after rt fw

Reply:
sorry I did misread the code here...